### PR TITLE
Update JWT validation guide to use default AS

### LINKED ISSUE #1208: Update dotnet JWT verification guide
https://developer.okta.com/code/dotnet/jwt-validation.html

- [x] Use default AS
- [x] Update screenshots to new dev console
- [x] Link to new ASP.NET Authentication Quick Start Guide at the top

### PR DESCRIPTION
**Pull request by nbarbettini**
## Description:
- Use the new `default` authorization server
- Remove old screenshot (no longer needed)
- Link to new .NET Auth Quick Start Guide

### Resolves:
Fixes #1208

### CODE REVIEW

**Comment by nbarbettini** [comment]
Ping @mraible for review + merge. (Doesn't need UAT, just a markdown sanity check)

File: _source/_code/dotnet/jwt-validation.md [hunk 1]
```diff
@@ -15,22 +15,20 @@ A common practice is to send one of these tokens in the `Bearer` header of futur
 
 You **don't** need to validate tokens manually if:
 
-* You are using ASP.NET or ASP.NET Core with the `JwtBearer` or `OpenIdConnect` middleware
+* You are using [ASP.NET](https://developer.okta.com/quickstart/#/widget/dotnet/aspnet4) or [ASP.NET Core](https://developer.okta.com/quickstart/#/widget/dotnet/aspnetcore) with the `JwtBearer` or `OpenIdConnect` middleware
```
Comment (mraible) [inline _source/_code/dotnet/jwt-validation.md]: These should be relative links, rather than absolute.

**Comment by mraible** [comment]
I don't have permission to merge. Just a minor change from absolute to relative URLs.

File: _source/_code/dotnet/jwt-validation.md (hunk 1 above)
Comment (nbarbettini) [reply _source/_code/dotnet/jwt-validation.md]: Ah, good catch.

**Comment by nbarbettini** [comment]
Ping @mystiberry-okta for merge

File: _source/_code/dotnet/jwt-validation.md [hunk 2]
```diff
@@ -15,22 +15,20 @@ A common practice is to send one of these tokens in the `Bearer` header of futur
...
 ## What you'll need
 
-* Your issuer URL
-* JWT (string)
+* Your authorization server URL
+* A JWT (string)
 * Libraries for retrieving the signing keys and validating the token
 
-The issuer URL is the URL of your Authorization Server (like `https://{yourOktaDomain}.com/oauth2/abc123`), which you can find in the Okta Developer Dashboard:
+When you create a [new Okta developer org](https://www.okta.com/developer/signup), Okta creates an authorization server called `default`. The URL for this authorization server is `https://{yourOktaDomain}.com/oauth2/default`. For example, if your Okta domain is `dev-1234.oktapreview.com`, the authorization server URL would be `https://dev-1234.oktapreview.com/oauth2/default`. If you want to use an authorization server you've created, you can copy the Issuer URL from that authorization server's details.
```
Comment (jmelberg-okta) [inline _source/_code/dotnet/jwt-validation.md]: This line is causing our travis test to fail:
`https://dev-1234.oktapreview.com`

Do you think this level of instruction is required? Can we remove the `For example, if your Okta...`?

Comment (mystiberry-okta) [reply _source/_code/dotnet/jwt-validation.md]: To be consistent with all the rest of the doc, yes. alternative is to write around the test:

"...replace {yourOktaDomain} with 'dev-1234.oktapreview' if the point is to keep them from adding -admin with is always a problem.

**Comment by nbarbettini** [comment]
This is currently waiting for me to rebase to weekly. Didn't realize I had to do that first.

**Comment by nbarbettini** [comment]
Rebased to `weekly` and resolved the bad example URL problem that @jmelberg-okta pointed out. I changed around some of the language (can be seen in the most recent commit). If you think it looks okay @mystiberry-okta, merge please! 😄

**Pull request merged**